tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderSummarySteps.cs

```diff
@@ -19,11 +20,20 @@ internal sealed class OrderSummarySteps
 
         private readonly string _orderSummaryUrl;
 
-        public OrderSummarySteps(Response response, Request request, Settings settings)
+        private OrderSummaryDataFactory _dataFactory;
+
+        public OrderSummarySteps(Response response, Request request, Settings settings, OrderSummaryDataFactory dataFactory)
         {
             _response = response;
             _request = request;
             _orderSummaryUrl = settings.OrderingApiBaseUrl + "/api/v1/orders/{0}/summary";
+            _dataFactory = dataFactory;
+        }
+
+        [Given(@"the user creates a new ""(.*)"" order with id (.*)")]
+        public async Task GivenTheDatabaseHasBeenInitialisedWithTheOrderData(string datasetKey, string orderId)
```

Comment:
Very minor but usually try & keep the method name similar to the step description.